src/generators/demoing-storybook-ts/templates/package.json

```diff
@@ -4,6 +4,6 @@
     "storybook:build": "tsc && build-storybook"
   },
   "devDependencies": {
-    "@web/dev-server-storybook": "^0.3.5"
+    "@web/dev-server-storybook": "next"
```

Comment:
Do we want this as next? Or should we move to a "stable" and or alpha version that can resolve up without getting ahead unexpectedly in conjunction with this?

Reply:
next for now, more changes in a subsequent pr